frontend/src/components/user-page/user-page.tsx

```diff
@@ -0,0 +1,42 @@
+import { useAppSelector } from 'hooks/hooks';
+import { RootState } from 'common/types/types';
+import styles from './styles.module.scss';
+
+const UserPage: React.FC = () => {
+  const { user } = useAppSelector(({ auth }: RootState) => ({
+    user: auth.user,
+  }));
+
+  console.warn(user);
```

Comment:
We should not call console in production

Reply:
I forgot to remove, thx